wp-content/civi-extensions/goonjcustom/Civi/GoonjInitiatedEventsService.php

```
@@ -38,6 +42,199 @@ public static function getSubscribedEvents() {
+
+    $eventId = $activity['Material_Contribution.Event'];
+    $contactId = (int) $activity['source_contact_id'];
+
+    $materialActivities = Activity::get(FALSE)
+      ->addSelect('source_contact_id')
+      ->addWhere('Material_Contribution.Event', '=', $eventId)
+      ->addWhere('id', '!=', $objectId)
+      ->execute();
+
+    $materialContactIds = [];
+    $currentContactStillExists = FALSE;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$currentContactStillExists'. (Unused Code Rules)

(UnusedLocalVariable)